sdk/bootstrap/build.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
Could do `/usr/bin/env bash`, although we're probably not portable across systems for many other more fundamental reasons.